// elgato-stream-deck/dist/models/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.StreamDeckOriginalMK2 = exports.StreamDeckOriginalV2 = exports.StreamDeckXLV2 = exports.StreamDeckXL = exports.StreamDeckMiniV2 = exports.StreamDeckMini = exports.StreamDeckOriginal = void 0;
__exportStar(require("./id"), exports);
var original_1 = require("./original");
Object.defineProperty(exports, "StreamDeckOriginal", { enumerable: true, get: function () { return original_1.StreamDeckOriginal; } });
var mini_1 = require("./mini");
Object.defineProperty(exports, "StreamDeckMini", { enumerable: true, get: function () { return mini_1.StreamDeckMini; } });
var miniv2_1 = require("./miniv2");
Object.defineProperty(exports, "StreamDeckMiniV2", { enumerable: true, get: function () { return miniv2_1.StreamDeckMiniV2; } });
var xl_1 = require("./xl");
Object.defineProperty(exports, "StreamDeckXL", { enumerable: true, get: function () { return xl_1.StreamDeckXL; } });
var xlv2_1 = require("./xlv2");
Object.defineProperty(exports, "StreamDeckXLV2", { enumerable: true, get: function () { return xlv2_1.StreamDeckXLV2; } });
var originalv2_1 = require("./originalv2");
Object.defineProperty(exports, "StreamDeckOriginalV2", { enumerable: true, get: function () { return originalv2_1.StreamDeckOriginalV2; } });
var original_mk2_1 = require("./original-mk2");
Object.defineProperty(exports, "StreamDeckOriginalMK2", { enumerable: true, get: function () { return original_mk2_1.StreamDeckOriginalMK2; } });
